lib/registration.pm

```diff
@@ -1075,7 +1075,10 @@ sub runtime_registration {
     if (is_transactional) {
         trup_call('register' . $cmd);
         trup_call('--continue run zypper --gpg-auto-import-keys refresh') if is_staging;
-        add_suseconnect_product('SL-Micro-Extras') if (is_sle_micro('>=6.0'));
+        if (is_sle_micro('>=6.0')) {
+            process_reboot(trigger => 1);
```

Comment:
Would you please explain why reboot is needed both before and after registration?

Reply:
@alice-suse Hi, to mitigate the issue with SLE Micro reported by @nanzhg essentially add_suseconnect_product('SL-Micro-Extras') doesn't work unless new snapshot is in use which requires soft reboot / full reboot